.eslintrc

```diff
@@ -1,3 +1,22 @@
 {
-  "extends": "google"
+  "extends": "google",
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "no-implicit-coercion": [2, {
```

Comment:
ha, they really need some constants for that. 0 is off, 1 is warning, 2 is error. 2 is the default for all of these, this just adds specific exceptions